testing/widgets/common/EditorAppearanceVariants.test.js

```diff
@@ -0,0 +1,46 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import asyncForEach from '../../../utils/visual-tests/helpers/async-for-each';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('Common.EditorAppearanceVariants')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
+    await t.resizeWindow(900, 600);
+  });
+
+runManualTest('Common', 'EditorAppearanceVariants', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test('EditorAppearanceVariants', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    const stylingModes = ['outlined', 'filled', 'underlined'];
+    const labelModes = ['static', 'floating', 'hidden'];
+
+    const clickSaveButton = async () => {
+      await t.click($('.validate.dx-button'));
+    };
+
+    const changeLabelMode = async (labelMode) => {
+      await t.click($('.dx-selectbox').nth(3));
+      await t.click($('.dx-item').withText(labelMode));
```

Comment:
Let's extract classes to separate const variables as follows

const SELECTBOX_CLASS = 'dx-selectbox';

await t.click($(`.${SELECTBOX_CLASS`).nth(3));